source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -113,7 +114,7 @@ def _extend_pt_sections_with_direct_path(pt_journey, dp_journey):
         pt_journey.sections.extend(dp_journey.journeys[0].sections)
 
 
-def _extend_journey(pt_journey, fallback_dp, fallback_extremity):
+def _extend_journey(pt_journey, fallback_dp, fallback_extremity, is_start_fallback):
```

Comment:
:sob: I so sad (maybe coz' my code was unclear)
This param is redundant with part of `fallback_extremity`, which is a PeriodExtremity https://github.com/CanalTP/navitia/blob/8cd17a0de9b61aad7ccc0056ac6856e96b09c8b0/source/jormungandr/jormungandr/utils.py#L446
Maybe it's not clear enough, though. But then we should refactor the code or document it to clarify it.

Reply:
Indeed. :+1: